### PR TITLE
Add clize recipe

### PR DESCRIPTION
**Pull request by proinsias**
- [x] od

### CODE REVIEW

**Comment by proinsias** [comment]
@conda-forge-admin, please rerender

**Comment by proinsias** [comment]
@conda-forge/help-python – I believe this is ready for review now.

File: recipes/clize/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f54dedcf6fea90a3e75c30cb65e0ab1e832760121f393b8d68edd711dbaf7187
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
```
Comment (npavlovikj) [inline recipes/clize/meta.yaml]: I think this recipe can be `noarch`, so please add `noarch: python` in the `build` section.

File: recipes/clize/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+requirements:
+  host:
+    - attrs >=19.1.0,<20
```
Comment (npavlovikj) [inline recipes/clize/meta.yaml]: Usually `python` and `pip` are the only dependencies that go to `host` - everything else should be in `run`.

File: recipes/clize/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - clize.tests
+
+about:
+  home: https://clize.readthedocs.io/en/3.0/
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: "Turn functions into command-line interfaces"
+  dev_url: https://github.com/epsy/clize
+
```
Comment (npavlovikj) [inline recipes/clize/meta.yaml]: You can also add `doc_url: https://clize.readthedocs.io/en/stable/` here.

File: recipes/clize/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    - sigtools >=2.0
+    - six
+
+test:
+  imports:
+    - clize
+    - clize.tests
+
+about:
+  home: https://clize.readthedocs.io/en/3.0/
```
Comment (npavlovikj) [inline recipes/clize/meta.yaml]: This probably should be `home: https://clize.readthedocs.io/en/stable/`.

**Comment by npavlovikj** [comment]
The recipe LGTM, thank you for contributing @proinsias 🎉

**Pull request merged**